modules/plugins/jboss-as-7/src/test/java/org/rhq/modules/plugins/jbossas7/itest/AbstractServerComponentTest.java

```diff
@@ -107,8 +107,10 @@ public void testServerAttributeValidation() throws Exception {
             + "start() method due to invalid baseDir.");
 
         // Change the expectedRuntimeProductName property
+        boolean exceptException = true;
```

Comment:
Not too much indeed. But without the typo you can  understand at first glance :)